lib/task-data/schemas/dell-racadm-catalog.json

```diff
@@ -1,16 +1,16 @@
 {
-    "copyright": "Copyright 2016, EMC, Inc.",
```

Comment:
I don't see a reason to change the whole indent of original file, RackHD code convention prefers 4 spaces rather than 2 spaces.